src/validate_i18n.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import re
+
+from flask import request
+from werkzeug.exceptions import abort
+from functools import wraps
+
+DEFAULT_YEAR = '2019'
+DEFAULT_LANG = 'en'
+SUPPORTED_YEARS = {
+    '2019': ('en', 'ja')
+}
+
+
+def validate_i18n(func):
```

Comment:
Let's rename this module to a generic validator decoupled from i18n, since year is not necessarily tied to i18n. And soon we'll also have to validate chapters and maybe even contributors.